Guide/Greedy/Algorithmic Crush/README.md

```diff
@@ -0,0 +1,84 @@
+# Algorithmic Crush
+This problem comes from [HackerRank](https://www.hackerrank.com/challenges/crush)
+
+Category: Greedy Algorithms
+
+Difficulty: Hard
+
+## Problem
+
+### Overview
+- Given a list of size N with elements initialized to zero
+- Perform M operations with parameters a (start), b (end), and k (value)
+    - Each operation adds the value of k to each element from position a (index a-1) to position b inclusively
+- Return the value of the element in the list with the greatest final value
+
+### Input Format
+- The first line contains two integers N and M separated by a space
+- The list will be of size N
+- The next M lines contain three integers a, b, and k (in this order) each separated by a space
+
+### Constraints
+- 3 <= N <= 10^7
+- 1 <= M <= 2 * 10^5
+- 1 <= a <= b <= N
+- 0 <= k <= 10^9
+
+### Output Format
+- A single line with the returned value
+
+## Algorithm
+
+### Overview
+The naive method to solve this problem involves adding the value of ‘k’ to each element in the list from ‘a’ to ‘b’ inclusively for each of the ‘M’ operations.
```

Comment:
Maybe move this to conclusion? You are not implemented the naive method (I hope), so this probably shouldn't be in the Algorithm Overview as this part should be focused on explaining to the user how the implemented algorithm works.

The conclusion would be a good spot to discuss alternative ways to solve the problem and discuss why they are/aren't as efficient as the implemented algorithm